modules/hcp-bm-hosts.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-disconnected/hcp-deploy-dc-bm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-bm-hosts_{context}"]
+= Creating bare metal hosts for the hosted cluster
+
+A _bare metal host_ is an `openshift-machine-api` object that encompasses physical and logical details so that it can be identified by a Metal3 Operator. Those details are associated with other Assisted Service objects, known as _agents_.
+
+.Prerequisites
+
+Before you create the bare metal host and destination nodes, you must create the virtual machines.
+
+.Procedure
+
+To create a bare metal host, complete the following steps:
+
+. Create a YAML file with the following information:
++
+Because you have at least one secret that holds the bare metal host credentials, you need to create at least two objects for each worker node.
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <hosted_cluster_name>-worker0-bmc-secret <1>
+  namespace: <hosted_cluster_namespace>-<hosted_cluster_name> <1> <2>
+data:
+  password: YWRtaW4=
```

Comment:
Remark here that this is the password of the BMC in Base64 format. 

This is how to get the password encoded in the right way:
 
`echo "admin" | base64 -w0`

Reply:
I added an annotation for the password line to instruct the user to specify the password of the BMC in Base64 format.